src/main/java/hw/spring/config/WebAppConfig.java

```diff
@@ -13,7 +13,7 @@
 public class WebAppConfig {
 
     @Bean
-    public InternalResourceViewResolver resolver() {
+    public InternalResourceViewResolver init() {
```

Comment:
```suggestion
    public InternalResourceViewResolver getResolver() {
```
так буде інформативніше